android/trifle/src/main/java/app/cash/trifle/KeyHandle.kt

```diff
@@ -45,34 +46,35 @@ data class KeyHandle internal constructor(val alias: String) {
     throw IllegalStateException(exceptionMessage)
   }
 
-  fun serialize(): ByteArray = alias.toByteArray(Charsets.UTF_8)
+  fun serialize(): ByteArray = tag.toByteArray(Charsets.UTF_8)
 
   companion object {
     // Throw an illegal state exception if we can't get hold of the proper key material. This
     // *should never happen* since the only way to obtain a KeyHandle is to deserialize one, which
     // should have already checked for this, or to generate a new one.
     private const val EXCEPTION_MSG =
-      "Android KeyStore does not contain a keypair corresponding to the %s alias"
+      "Android KeyStore does not contain a keypair corresponding to the %s tag"
     private const val ANDROID_KEYSTORE_TYPE: String = "AndroidKeyStore"
     private val KEY_STORE = KeyStore.getInstance(ANDROID_KEYSTORE_TYPE).apply {
       load(null)
     }
 
     fun deserialize(bytes: ByteArray): KeyHandle {
-      val alias = bytes.toString(Charsets.UTF_8)
-      if (!containsAlias(alias)) {
-        throw IllegalStateException(String.format(EXCEPTION_MSG, alias))
+      val tag = bytes.toString(Charsets.UTF_8)
+      if (!containsTag(tag)) {
+        throw IllegalStateException(String.format(EXCEPTION_MSG, tag))
       }
-      return KeyHandle(alias)
+      return KeyHandle(tag)
     }
 
-    //TODO(dcashman): Consoidate API surface with iOS surface.
-    internal fun generateKeyHandle(alias: String): KeyHandle = KeyHandle(alias)
+    //TODO(dcashman): Consolidate API surface with iOS surface.
```

Comment:
do we still need this comment?

Reply:
Yea keeping it for now since there are some slight differences